### ISSUE TITLE
Issue with Android 10 Devices

### ISSUE THREAD

**Issue by lekowan** [vilic/cordova-plugin-tts#115]
Is anyone else experiencing issues with Android 10 (particularly Samsung devices) only? The plugin doesn't seem to read out content in the target language. Thanks in advance.

**Comment by surajbhadola01** [comment]
Have you find any solution I am also facing same problem

**Comment by lekowan** [comment]
Yes, I have. Latest Samsung devices use the Samsung TTS engine by default which doesn't work well. It needs to be switched to the Google TTS engine in the device settings. There is also a way to force the device to use Google TTS, which I implemented in my Ionic, but it's been a while and I totally forgot how I did that.

**Comment by lorenzpfei** [comment]
> Yes, I have. Latest Samsung devices use the Samsung TTS engine by default which doesn't work well. It needs to be switched to the Google TTS engine in the device settings. There is also a way to force the device to use Google TTS, which I implemented in my Ionic, but it's been a while and I totally forgot how I did that.

May you check what you did to fix this? I'm despairing on this.
